components/Share.vue

```diff
@@ -53,7 +53,7 @@
           class="g-button"
           no-caps
           color="primary"
-          :disable="loading || (emptyResponse && query.type !== 'DIDAuth')"
+          :disable="loading || (emptyResponse && query.type !== 'DIDAuthentication')"
```

Comment:
Let's make this a backwards compatible change (and use the computed var):

```suggestion
          :disable="loading || (emptyResponse && headerType !== 'authentication')"
```